src/Component/Tests/Dummy/AttributeFooGrid.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Tests\Dummy;
+
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Component\Grid\Attribute\AsGrid;
+
+#[AsGrid(resourceClass: Foo::class)]
+final class AttributeFooGrid extends AbstractGrid
```

Comment:
Is it required to extend `AbstractGrid` to take advantage of `AsGrid` attribute?

Reply:
Nice catch, I will take a loot. AFAIK an interface should be enough

Reply:
So, the AbstractGrid fetches the name in attributes or via code.
If we use GridInterface, getName() and toArray() must be reimplemented.